packages/meteor/debug.js

```diff
@@ -39,6 +39,11 @@ Meteor._debug = function (/* arguments */) {
 
         if (allArgumentsOfTypeString)
           console.log.apply(console, [Array.prototype.join.call(arguments, " ")]);
+        // We want to print out the stack trace when dynamics_nodejs.js throws
+        // an exception. Need to know what case this is and then make it into
+        // array so that we can pass it to console.log.apply.
+        else if (arguments.length === 2)
+          console.log.apply(console, [arguments[1]]);
```

Comment:
I'm just confused about why you're editing `Meteor._debug` at all.

`Meteor._debug` is a generic function used all over the place.  "Ignore the first argument if there are two arguments and they are not both strings" seems to be incredibly surprising behavior to me.

Can you describe the path that this exception you're worried about takes in the current code from it being through to it being printed poorly?